plat/drivers/include/virtio/virtio_bus.h

```diff
@@ -268,25 +268,25 @@ static inline int virtio_config_get(struct virtio_dev *vdev, __u16 offset,
  * The helper function to find the number of the vqs supported on the device.
  * @param vdev
  *	A reference to the virtio device.
- * @param total_vqs
- *	The total number of virtqueues requested.
+ * @param vq_id
+ *	The virtqueue id whose size needs to be found out
+ *	(0 to max_virqueues - 1 inclusive)
  * @param vq_size
- *	An array of max descriptors on each virtqueue found on the
- *	virtio device
+ *	A reference to the virtq size, size of virt queue is stored in it
  * @return int
- *	On success, the function return the number of available virtqueues
+ *	On success, the function return 0
```

Comment:
Both `vpci_legacy_pci_vq_find` and `vm_find_vqs` return 1 at the moment.

Reply:
yes, you are right. I will update my comment.